gtsam/geometry/tests/testPose3.cpp

```diff
@@ -145,6 +145,75 @@ TEST(Pose3, Adjoint_full)
   EXPECT(assert_equal(expected3, Pose3::Expmap(xiprime3), 1e-6));
 }
 
+/* ************************************************************************* */
+// Check Adjoint numerical derivatives
+TEST(Pose3, Adjoint_jacobians)
+{
+  Matrix6 actualH1, actualH2, expectedH1, expectedH2;
+  std::function<Vector6(const Pose3&, const Vector6&)> Adjoint_proxy =
+      [&](const Pose3& T, const Vector6& xi) {
+        Vector6 res1 = T.AdjointMap() * xi, res2 = T.Adjoint(xi);
+        EXPECT(assert_equal(res1, res2));
+        return res1;
+      };
+
+  gtsam::Vector6 xi =
```

Comment:
Use the syntax `xi(1, 2, 3, 4, 5, 6)`?

Reply:
Thanks!  For some reason I thought this only worked up to Vector3 but good to know I was just totally wrong :)

Reply:
Actually I'm having trouble getting this to work - do you have a reference?

I can't seem to get the initializer-list constructor to work for some reason:

[Documentation](https://eigen.tuxfamily.org/dox/group__TutorialMatrixClass.html)
> Matrices and vectors can also be initialized from lists of coefficients. Prior to C++11, this feature is limited to small fixed-size column or vectors up to size 4:
...
> If C++11 is enabled, fixed-size column or row vectors of arbitrary size can be initialized by passing an arbitrary number of coefficients:
... (brace initializer lists)

Trying
```c++
gtsam::Vector6 xi {0.1, 1.2, 2.3, 3.1, 1.4, 4.5};
gtsam::Vector6 xi = {0.1, 1.2, 2.3, 3.1, 1.4, 4.5};
gtsam::Vector6 xi {{0.1, 1.2, 2.3, 3.1, 1.4, 4.5}};
```
but none seem to be working :(

Reply:
Update: I think we must be on an old version of Eigen that doesn't have this initializer - the [latest Eigen Matrix.h file](https://eigen.tuxfamily.org/dox/Matrix_8h_source.html) Lines 286-322 have the initializer_list code but in our 3rdparty folder those lines are missing.

In light of this, I'm not going to change these lines, ok?

Please resolve if this is ok with you.

Reply:
just for posterity (my own notes) initializer list constructor wasn't added until at least 3.3.8, started by this commit 2 years ago: https://gitlab.com/libeigen/eigen-backup/-/commit/db152b9ee6effd3799f70a621f495c427cb3c33f

3.3.7 is 3 years old now.

Reply:
Use parentheses not curly braces. That works perfectly.

Reply:
didn't work for me, is this GTSAM custom constructor or eigen?

Can you confirm that you have tried compiling?  I spent 2 hours this morning trying to make this work by your suggestion and would rather not spend 2 more before having the peace of mind that you confirmed it working on your machine.